### ISSUE TITLE
mPLUG-HiTeA-视频描述-英文-Base示例代码出现TypeError: VideoCaptioningPipeline: HiTeAForAllTasks: 'NoneType' object is not callable错误

### ISSUE THREAD

**Issue by zhougr18** [modelscope/modelscope#265]
在运行以下mPLUG-HiTeA-视频描述-英文-Base示例代码时出错
![image](https://user-images.githubusercontent.com/61300157/230890102-3ee50dfe-c660-4cb2-b041-873520eac60d.png)
报错信息如下
![image](https://user-images.githubusercontent.com/61300157/230890328-f6081b7b-d06f-41d6-9826-2bd69fb847ec.png)
请问该如何解决

**Comment by ChandanVerma** [comment]
@zhougr18  Couldyou please provide the resolution for this? I am getting the same issue... Any help would be deeply appreciated.
Thanks

**Comment by zhougr18** [comment]
@ChandanVerma I found the reason is that there is no fairscale package in my environment, so I solved the problem by installing fairscale. Specifically, there is a try-except statement located atmodelscope/models/multi_modal/mplug/mvit.py 14-17 lines to import fairscale.nn.checkpoint as checkpoint_wrapper which assigned checkpoint_wrapper to None. And then the None value of checkpoint_wrapper caused that HiTeAForAllTAsks was None.
![image](https://user-images.githubusercontent.com/61300157/233892516-1ccf96c8-0f9d-47ed-87e6-4a06a8b2285c.png)
